R/Design-methods.R

```diff
@@ -417,6 +417,19 @@ setMethod("simulate",
           data = thisData
         )
 
+        # Get the MTD estimate from the samples.
+        sample_mtd <- dose(
+          mean(object@nextBest@target),
+          model = object@model,
+          samples = thisSamples
+        )
+
+        # Create list with median MTD and CV.
+        additional_stats <- list(
```

Comment:
this is nice I think, however I wonder if we can make this more flexible for the user - e.g. how about accepting a list of functions as part of the `simulate()` call and then the user can define `median`, `cv` functions etc. to apply to `target_dose_samples`?

Reply:
Great proposal. I really like that idea. This will add much flexibility e.g. if someone want to see the mean instead of the median etc.. We just have to evaluate, if it is straight forward to implement.

Reply:
@danielinteractive @0liver0815 Sure, as the next step, I will try to implement the functions where user can define median, cv, mean etc. functions.